### PR TITLE
Contacts listing 

### PR DESCRIPTION
**Pull request by johangant**

### CODE REVIEW

**Comment by johangant** [comment]
This PR provides the config needed to provide the contacts feature's views displays and switch on the custom module controlling the section

File: config/sync/views.view.contacts.yml [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+          field_rendering: true
+          fallback_handler: search_api
+          fallback_options:
+            link_to_item: false
+            use_highlighting: false
+            multi_type: separator
+            multi_separator: ', '
+          plugin_id: search_api_field
+      filters:
+        type:
```
Comment (DuttonMa) [inline config/sync/views.view.contacts.yml]: Should these view be filtered on 'published' - or will the search already have restricted the results to only published contacts ?

File: config/sync/core.extension.yml [hunk 2]
```diff
@@ -56,10 +56,12 @@ module:
   metatag_twitter_cards: 0
   metatag_verification: 0
   migrate: 0
+  migrate_drupal: 0
   migrate_plus: 0
```
[the remark this answers is not in the document]
Comment (johangant) [reply config/sync/core.extension.yml]: You're right, let me create a new PR to pull these out

File: config/sync/views.view.contacts.yml (hunk 1 above)
Comment (johangant) [reply config/sync/views.view.contacts.yml]: Good question. Search API already uses a processor plugin to do this so I'm relying on that to do the hard work for me in this instance. You'll note on the node based view (for the A-Z) I'm filtering for published contacts

**Comment by johangant** [comment]
@DuttonMa thanks for the feedback. Your question about cache tags is definitely valid... given that it'll be applicable to this and all future listings we define, I'd support adding those cache tag modules.

If you don't mind, please could you set up a new Jira ticket to track this? That way you (or any of us) can get this added with any existing views tweaked to use the most appropriate cache tags

**Comment by DuttonMa** [comment]
Yes, I'll do that now. I think that this is the first user facing view taht we have implemented but certainly worth checking.

File: config/sync/core.extension.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (johangant) [reply config/sync/core.extension.yml]: There it is now: https://github.com/dof-dss/nidirect-drupal/pull/18

**Pull request merged**